content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -427,8 +427,12 @@ or updating objects that contain Pod templates, such as Deployments, Jobs, State
 See [Enforcing Pod Security at the Namespace Level](/docs/concepts/security/pod-security-admission)
 for more information.
 
-## seccomp.security.alpha.kubernetes.io/pod and container.seccomp.security.alpha.kubernetes.io/[NAME] (deprecated)
+## seccomp.security.alpha.kubernetes.io/pod (deprecated)
 
-The seccomp annotations have been deprecated since Kubernetes v1.19 and will
-become non-functional in v1.25. Please use the `seccompProfile` of the
-`SecurityContext` instead.
\ No newline at end of file
+The Seccomp annotation has been dperecated since Kubernetes v1.19 and will become non-functional in v1.25.
+To specify security settings for a Pod, include the `securityContext` field in the Pod specification. The `securityContext` field is a [PodSecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#podsecuritycontext-v1-core) object. The [security settings](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod) that you specify for a Pod apply to all Containers in the Pod.
+
+## container.seccomp.security.alpha.kubernetes.io/[NAME] (deprecated)
+
+The Seccomp annotation has been deprecated since Kubernetes v1.19 and will become non-functional in v1.25.
+Now `seccomp` is a class inside of the pod or container. This can be [changed by](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container) using `type` of the `seccompProfile` of the `securityContext`.
```

Comment:
```suggestion
Now `seccomp` is a class inside of the pod or container. This can be
[changed by](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container)
using `type` of the `seccompProfile` of the `securityContext`.
```